USWDS - Utilities: Update inaccurate comments

amyleadem opened the pull request:
# Summary

**Updated the code comments on utility Sass partials.** These comments now reflect the correct utility class names and values.

> [!Important]
> This PR builds on the changes added by @aduth in PR #5700. Thank you @aduth!

## Breaking change

This is not a breaking change. This update only affects code comments.

## Related issue

Closes #5860 

## Related pull requests

This PR builds on the changes from @aduth in PR #5700.

## Preview link

N/A

## Problem statement

The code comments on the files in the `packages/uswds-utilities/src/styles/rules/` directory are outdated and inaccurate. These code comments should be updated so that developers can use the code comments as reference (especially for utilities not documented on our website).

## Solution

This PR updates outdate utility class references to match uswds 3.0 naming conventions. 

## Testing and review
1. Confirm that all files in the `packages/uswds-utilities/src/styles/rules/` have comments that reflect current USWDS naming structures
1. Confirm comments are formatted consistently

mejiaj commented on packages/uswds-utilities/src/styles/rules/add-aspect.scss:
```diff
@@ -3,16 +3,17 @@
+    height: inherit;
+    padding: inherit;
+    aspect-ratio: 1.7777777778;
+    max-width: 100%; }
```
Love the improved comments.

Small change to indent to match standards below.

```suggestion
    max-width: 100%; 
  }
```

amyleadem replied:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

mejiaj commented on packages/uswds-utilities/src/styles/rules/add-list-reset.scss:
> [!TIP]
> Not alphabetical order, but it's output in `uswds.css`.

```css
/* uswds.css */
.add-list-reset{
  margin-bottom:0;
  margin-top:0;
  padding-left:0;
  list-style:none;
}
```

mejiaj commented on packages/uswds-utilities/src/styles/rules/background-color.scss:
```diff
@@ -9,8 +9,8 @@ output:
-  .bg-red-50v {
-    background-color: #be4900; }
+  .bg-red {
+    background-color: #e52207; }
```
```suggestion
    background-color: #e52207; 
  }
```

amyleadem replied:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

mejiaj commented on packages/uswds-utilities/src/styles/rules/border-color.scss:
```diff
@@ -1,16 +1,16 @@
 example:
   .border-red {
-    border-color: #be4900; }
+    border-color: #e52207; }
```
```suggestion
    border-color: #e52207; 
  }
```

amyleadem replied:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

mejiaj commented on packages/uswds-utilities/src/styles/rules/border-radius.scss:
```diff
@@ -3,17 +3,17 @@
-  .border-radius-pill {
-    border-radius: 10em; }
+  .radius-pill {
+    border-radius: 99rem; }
```
```suggestion
    border-radius: 99rem; 
  }
```

amyleadem replied:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

mejiaj commented on packages/uswds-utilities/src/styles/rules/border.scss:
```diff
@@ -12,11 +12,11 @@ output:
 
   .border-0 {
-    border: none; }
+    border: 0 solid; }
```
```suggestion
    border: 0 solid; 
  }
```

amyleadem replied:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

mejiaj commented on packages/uswds-utilities/src/styles/rules/border.scss:
```diff
@@ -12,11 +12,11 @@ output:
 example:
-  .border-t-1px {
+  .border-top-1px {
     border-top: 1px solid; }
```
```suggestion
    border-top: 1px solid; 
  }
```

mejiaj commented on packages/uswds-utilities/src/styles/rules/overflow.scss:
```diff
@@ -3,10 +3,10 @@
 ----------------------------------------
 usage:
-  .overflow-[modifier]
+  .overflow-[modifier]*-[key]
```
What's the value of the `*`?

amyleadem replied:
I _believe_ it indicates that the modifier is optional. At least, that is my assumption based on the other comments with this format. I read it as the class can can be written as `.overflow-hidden` or `.overflow-y-hidden`.

mejiaj reviewed (changes requested):
Thanks for making these fixes. I was able to test comments are accurate and up-to-date. 

Can we fix the closing `}` in the code examples?

mahoneycm commented on packages/uswds-utilities/src/styles/rules/opacity.scss:
```diff
@@ -11,6 +11,9 @@ output:
   .opacity-0 {
     opacity: 0; }
+
+  .opacity-30 {
+    opacity:0.3; }
```
Missing a space 
```suggestion
  .opacity-30 {
    opacity: 0.3; }
```

amyleadem replied:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)

mahoneycm reviewed (changes requested):
This is looking great, thank you for combing through these and updating.

Besides James' new line comment, I noticed a space missing for one value. Minuscule change but might as well while we're here 😄

amyleadem commented:
@mejiaj and @mahoneycm  Thanks for those comments. I've moved the closing braces in the comments to a new line to meet standards and added the missing space to the opacity comment. Please let me know if I missed anything!

thisisdano reviewed (approved):
Thanks @aduth and @amyleadem! A somewhat thankless but important task

thisisdano commented:
But now it's thankless no longer :)
